features/SideNavigation.feature

```diff
@@ -0,0 +1,165 @@
+            | Child                                       | 003B45 (dark green) | FFFFFF              | bold       | N/A         | dropdown line changes to 2px width and green (06966F) | N/A              | corresponding content page opening up |
+            | Child with dropdown arrow                   | FFFFFF              | 003B45 (dark green) | bold       | N/A         | 003B45 (dark green)                                   | arrow point up   | N/A                                   |
+
+    @ui
+    Scenario: When a page is not found a custom 404 error page is displayed
+        Given the user clicks on a link which is not available
+        Then they see a 404 Page Not Found message
+        And they see a custom design with Squatchy sitting in the middle of the 0
+        And a message:
+            """Please excuse the mess—We are in the process of making our documentation even better.
+    If you cannot find the information you're looking for, let our Success team know, and they will be more than happy to provide!"""
+
+    @ui
+    Scenario: The logo at the top of the side navbar takes you to the "https://docs.saasquatch.com/" page
+        Given the user is on any page of the docs portal
+        When they click the SaaSquatch logo at the top of the navigation bar
+        Then the navigation bar has every dropdown closed
+        And shows the 6 main menu items
+        And the central section of the page shows the "help center" search page
+        And the url is "https://docs.saasquatch.com/"
+
+    Scenario: There is a search box at the top of the side navbar (more specs available in a separate file for Search)
+        Given the user is on the SaaSquatch docs page at any location
+        Then a search input field is shown in the sidebar
+        And the default state for the search input field is empty
+        And the search is working expected
+
+    Scenario: In the side navbar there are different menu item types
+        Given the user is on the docs portal
+        Then they see the following side navbar item types
+
+            | types                                                    |
+            | Core Category Section with navigation arrow              |
+            | Core Category Section with icon only                     |
+            | Navigation item with an arrow down                       |
+            | Article item                                             |
+            | Core category item at the top of the navbar with an icon |
+
+    Scenario: The Core Category Section with navigation arrow takes you to another level of navigation items and opens up a page
+        Given the side navbar is in the starting state
+        When the user clicks on any part of a " Core Category Section with navigation arrow"
+        Then they see that in the nav bar a "Core category item at the top of the navbar with an icon" shows up
```

Comment:
Is it in a selected state? how does this top level behave when another doc is selected from the sub categories? Please add a bit more detail (either here or in another spec)